core/src/apps/cardano/get_address.py

```diff
@@ -65,37 +53,17 @@ async def _display_address(
     address: str,
     protocol_magic: int,
```

Comment:
Well, we could perhaps rename `protocol_magics.to_ui_string`with `protocol_magics.to_network_name` and the function could just take in a `network_name` as the common Trezor function `show_address` does. I don't think it's that much better though.

Reply:
I guess I am slightly in favor of the change.

Reply:
Hm, turns out I would need to move the `if not protocol_magics.is_mainnet(protocol_magic):` logic from `show_cardano_address` into to `_display_address` function and it IMHO doesn't belong there. I don't want to drop the logic since it doesn't bother ordinary users with showing the network name to them if it's mainnet.